src/handle/command.ml

```diff
@@ -555,10 +554,15 @@ let get_proof_data : compiler -> sig_state -> p_command -> cmd_output =
       in
       if p_sym_prf = None && not (finished pdata_state) then wrn pos
         "Some metavariables could not be solved: a proof must be given";
+      let ffff = match pe.elt with
+      | P_proof_abort -> Format.ifprintf Stdlib.(!(Stdlib.ref Format.std_formatter)) ("%s" ^^ "@.") "Print nothing!"
+      | P_proof_admitted -> Console.out 2 (Color.red "symbol %a : %a") uid id term a
+      | P_proof_end -> Console.out 2 (Color.red "symbol %a : %a") uid id term a
+      in ffff;
```

Comment:
Prefer to write:
```
begin
  match pe.elt with
  ...
end;
````

Reply:
Note that `Stdlib.(!(Stdlib.ref Format.std_formatter))` is the same as `Format.std_formatter`. `ref` creates a pointer to something. `!` returns the thing a pointer points to.
But why not simply writing:
```
| P_proof_abort -> ()
```
?

Reply:
Another remark: please check CONTRIBUTING.md and use misc/git_hook_helper.sh.